src/commands/call.ts

```diff
@@ -93,7 +115,65 @@ export class ActorCallCommand extends ApifyCommand<typeof ActorCallCommand> {
 			type: 'Actor',
 			waitForFinishMillis,
 			inputOverride: inputOverride?.input,
+			silent: this.flags.silent,
 		});
+
+		if (this.flags.outputDataset) {
+			// TODO: cleaner way to do this
+			while (!TerminalStatuses.includes(run.status as never)) {
+				run = (await apifyClient.run(run.id).get())!;
+
+				if (TerminalStatuses.includes(run.status as never)) {
+					break;
+				}
+
+				// Wait a second before checking again
+				await new Promise((resolve) => setTimeout(resolve, 1000));
+			}
+
+			const datasetId = run.defaultDatasetId;
+
+			let info: Dataset;
+			let retries = 3;
+
+			do {
+				info = (await apifyClient.dataset(datasetId).get())!;
+
+				if (info?.itemCount) {
+					break;
+				}
+
+				await new Promise((resolve) => setTimeout(resolve, 1000));
+			} while (retries--);
+
+			let actualView = this.flags.view;
+
+			// TODO: apify-client doesn't have this field yet
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			const schema = (info as any)?.schema;
+
+			if (schema?.views) {
+				// Ensure the view provided matches
+				if (actualView && !Reflect.has(schema.views, actualView)) {
+					if (!this.flags.silent) {
+						warning({
+							message: `View "${actualView}" does not exist in the dataset schema. Available views are: ${Object.keys(
+								schema.views,
+							).join(', ')}. Will print the dataset in the default view.`,
```

Comment:
```suggestion
							).join(', ')}. 
Will print the dataset in the default view.`,
```

Would fancy a linebreak here, but this is definitely a nit. Also, can the view names contain commas or spaces? If so, I'd rather wrap the _Available views_ names in quotes.